src/components/Hexagon/lookup.tsx

```diff
@@ -0,0 +1,50 @@
+export const hexColors = {
```

Comment:
There is no React here. You can rename the file to `lookup.ts`

Reply:
Done.